sql/index/pilosa/driver.go

```diff
@@ -166,6 +166,10 @@ func (d *Driver) Save(ctx context.Context, i sql.Index, iter sql.IndexKeyValueIt
 		return err
 	}
 
+	// make sure we delete the index in every run before inserting, since there may
+	// be previous data
+	_ = d.client.DeleteIndex(pilosaIndex)
```

Comment:
why isn't it necessary check the error if the index must be deleted?

Reply:
Done